storageClient/api.go

```diff
@@ -73,12 +73,31 @@ type AddingCapabilityRequest struct {
 	AccountID      uuid.UUID `json:"account_id"`
 }
 
-// AddingClientToGroupRequest wraps values used to add a client to a Group.
-type AddingClientToGroupRequest struct {
-	ClientID       uuid.UUID `json:"client_id"`
-	Name           string    `json:"group_name"`
-	CapabilityName string    `json:"capability_name"`
-	AccountID      uuid.UUID `json:"account_id"`
+// NewMember wraps values needed to add a client to a Group.
```

Comment:
comment needs to be updated for the struct name below and to generalize this to `GroupMembers wraps values for a client that is a member of a group ` as this struct is used and returned as part of adding/removing and listing group members.